forwards_paper.tex

```diff
@@ -537,7 +551,7 @@ \subsection*{Recording the pedigree in forwards time}
 }
 
 \algstep{W3.}{Choose parents.}{Set $a \leftarrow \randomuniform(\{0, \dots, N - 1\})$,
-    $b \leftarrow \randomuniform(\{0, \dots, N - 1\})$ and $x \leftarrow \randomuniform([0, L))$.
+    $b \leftarrow \randomuniform(\{0, \dots, N - 1\})$ and $x \leftarrow \randomuniform((0, L))$.
```

Comment:
Needs to be closed interval, or we can generate an empty edge (0, 0, p, c)

Reply:
good point.